### PR TITLE
Initial commit of composer.json

### PR DESCRIPTION
**Pull request by baldwinlouie**
Added a composer.json to build the Façade site.  This is pretty much the same as docomoinnovations/cloud_orchestrator.  But I think it is good to split this out so in the future we can do additional things without affecting the `cloud_orchestrator` composer.json.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    "drush/drush": "^11"
+  },
+  "require-dev": {
+    "dmore/behat-chrome-extension": "*",
+    "docomoinnovations/drupal-extension": "*",
+    "drupal/coder": "*"
+  },
+  "autoload-dev": {
+  }
+}
```
Comment (naoi) [inline composer.json]: @baldwinlouie Please add CRLF.  Thanks

**Review by naoi — CHANGES_REQUESTED** [review]
@baldwinlouie Thank you for the PR.  I posted my minor comment.  Thanks

**Comment by baldwinlouie** [comment]
@naoi I've added the CLRF at the end of the file.

**Review by naoi — APPROVED** [review]
@baldwinlouie Thank you for the PR.  I'll merge it.  Thanks!

**Pull request merged**